etc/membership_crontab

```diff
@@ -0,0 +1,3 @@
+# Drop this into /etc/cron.d
```

Comment:
Or just append to /etc/crontab - either works. It's more of an example than anything